src/styles/StyleUtils.js

```diff
@@ -337,8 +337,13 @@ function getLoginPagePromoStyle() {
             backgroundColor: colors.blue,
             backgroundImageUri: `${CONST.CLOUDFRONT_URL}/images/homepage/brand-stories/freeplan_blue.svg`,
         },
+        {
+            backgroundColor: colors.floralwhite,
+            backgroundImageUri: `${CONST.CLOUDFRONT_URL}/images/homepage/brand-stories/cpa-card.svg`,
+            redirectUri: `${CONST.USE_EXPENSIFY_URL}/accountants`,
```

Comment:
Hi! 👋  Coming in super late, but I noticed this and `StyleUtils` should only be returning things that are styles. This URL is used only for redirecting the user so it should not be returned from the styles. @ctkochan22 can you please fix this so that `Link.openExternalLink(backgroundStyle.redirectUri);` uses `${CONST.USE_EXPENSIFY_URL}/accountants` directly?